tests/acceptance/features/apiFullTextSearch/search.feature

```diff
@@ -25,3 +25,22 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search restored files by content
+    Given using <dav-path-version> DAV path
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has uploaded file with content "hello world from nepal" to "keywordAtStart.txt"
+    And user "Alice" has uploaded file with content "saying hello to the world" to "uploadFolder/keywordAtMiddle.txt"
```

Comment:
is there any specific reason to upload file inside folder otherwise we could add this step in background

Reply:
IMO, these two steps can go to background
`Given using <dav-path-version> DAV path`
`And user "Alice" has uploaded file with content "hello world from nepal" to "keywordAtStart.txt"`

Reply:
There is not able to put a step containing an example table in the background `Given using <dav-path-version> DAV path`
But 
I will put this upload file inside folder step in background `And user "Alice" has uploaded file with content "hello world from nepal" to "keywordAtStart.txt"`